### PR TITLE
Add Mesher into Service Mesh Landscape

### PR DESCRIPTION
**Pull request by yogi4**
Added Mesher and Reordered Rotor  Github issue 
https://github.com/layer5io/layer5/issues/10

### CODE REVIEW

File: _data/categories/meshes.yml [hunk 1]
```diff
@@ -69,6 +59,27 @@ Open Source. From Buoyant. Written in Rust and Go."
   multi-cluster: "?"
   multi-tenant: "?"
 
+- name: Mesher
+  desc: "A service mesh implementation based on go chassis. One big advantage of Mesher is it is able to work with go-chassis in same service mesh control plane like Istio. Without Istio they can work together with ServiceComb Service center and running on any infrastructure(docker, VM, baremetal).
```
Comment (leecalcote) [inline _data/categories/meshes.yml]: Even thought the project's README isn't (https://github.com/go-chassis/go-chassis), let's be consistent with how we reference go-chassis. There are two different grammatical styles used in these couple sentences.

Let's avoid opinions, adjectives and comparisons/analogies in the description of the projects in attempt to provide unbiased depictions focused on facts. As examples, let's avoid "big advantage", "like Istio", "easily".

"docker" isn't an infrastructure.

Let's do what we can to cut down the length of descriptions here given that they are shown in tooltips - <a href="https://ibb.co/YDSgG1d">Screen-Shot-2019-01-17-at-6-16-09-AM</a>

File: _data/categories/meshes.yml [hunk 2]
```diff
@@ -69,6 +59,27 @@ Open Source. From Buoyant. Written in Rust and Go."
...
+Mesher supports both linux and windows OS, that makes possible that .Net service can work with java, go, python service in one distributed system easily"
+  link: https://github.com/go-mesh/mesher
+  autoinject: "Yes"
+  h2: "Yes"
+  grpc: "Yes"
+  tcp-web: "No"
+  multi-cluster: "Yes, Powered by ServiceComb"
```
Comment (leecalcote) [inline _data/categories/meshes.yml]: Lowercase "p" in Powered.

File: _data/categories/meshes.yml [hunk 3]
```diff
@@ -69,6 +59,27 @@ Open Source. From Buoyant. Written in Rust and Go."
...
+  multi-tenant: "Huawei cloud only"
```
Comment (leecalcote) [inline _data/categories/meshes.yml]: Capital "C" in cloud.

File: _data/categories/meshes.yml (hunk 1 above)
Comment (yogi4) [reply _data/categories/meshes.yml]: "A service mesh implementation based on go-chassis, Mesher can work with go-chassis in same service mesh control and can work with ServiceComb Service center running on any infrastructure. Mesher supports both linux and windows OS"

Thank you  Lee good Insights. Does the above condensed statement seem fair and unbiased.

Comment (yogi4) [reply _data/categories/meshes.yml]: @leecalcote  this is the updated version . "Mesher is service mesh implementation based on go-chassis which can work together with ServiceComb Service center running on any infrastructure. Mesher can work with go-chassis in same service mesh control plane and it supports both linux and windows OS"

**Comment by leecalcote** [comment]
@yogi4 good deal.

**Pull request merged**